example/styleguide.json

```diff
@@ -1,5 +1,9 @@
 {
   "title": "React Style Guide",
+  "typekit": {
+    "id": "rtq7sdx",
```

Comment:
Ya. I'd go with having an example in the README. Not it the actual live example, because there are other other ways to add fonts